packages/schematics/angular/library/files/tsconfig.lib.json.template

```diff
@@ -15,8 +15,6 @@
     "annotateForClosureCompiler": true,
     "skipTemplateCodegen": true,
     "strictMetadataEmit": true,
```

Comment:
Note: the below options are not needed by ngtsc compiler, but we will leave them to make opt-out easier.
```
skipTemplateCodegen: true,
strictMetadataEmit: true,
enableResourceInlining: true,
```